qiita_client/qiita_client.py

```diff
@@ -714,3 +743,116 @@ def _process_files_per_sample_fastq(self, files, prep_info,
         prep_info = prep_info.filter(items=sample_names.keys(), axis=0)
 
         return sample_names, prep_info
+
+    def fetch_file_from_central(self, filepath, prefix=''):
```

Comment:
Wondering if the default should be for prefix to be None? Maybe this is clear that it will be ignored ...

Reply:
agreed. I thought testing for this could be done via `If not prefix:`, however, this does not seem to work with `None`?! Therefore, if conditions are slightly longer now